ui.tests/test-module/specs/emailinput/emailinput.authoring.spec.js

```diff
@@ -74,6 +74,26 @@ describe('Page - Authoring', function () {
             testEmailInputBehaviour(emailInputEditPathSelector, emailInputDrop);
         });
     });
+
+    it('Invalid email input should show error message', { retries: 3 }, function () {
+      dropEmailInputInContainer();
+      cy.get('[data-layer="Preview"]').click();
+      getPreviewIframeBody().find('input.cmp-adaptiveform-emailinput__widget')
+          .should('be.visible').type('invalidEmail', {force: true}).blur({force: true});
+      getPreviewIframeBody().find('.cmp-adaptiveform-emailinput__errormessage').should('exist');
```

Comment:
This test needs to be written in emailinput.runtime.spec.js. Let's not access iframe details inside authoring